actionview/test/activerecord/partial_rendering_query_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "active_record_unit"
+
+class PartialRenderingQueryTest < ActiveRecordTestCase
+  def setup
+    @view = ActionView::Base
+      .with_empty_template_cache
+      .with_view_paths(ActionController::Base.view_paths, {})
+
+    @queries = []
+
+    ActiveSupport::Notifications.subscribe("sql.active_record") do |*, payload|
```

Comment:
You should unsubscribe to the notification after this test, or use a `subscribed` block to automatically unsubscribe. Otherwise, all subsequent tests will push to this array.